### ISSUE TITLE
What shells does it support

### ISSUE THREAD

**Issue by 0323pin** [shosatojp/messa#22]
Hi,
Curious, does it only work with `bash`?
Or, does it support other shells?

I'm interested to try it with `nu`.
Thx!

**Comment by shosatojp** [comment]
I released v2.1.0 and zsh and fish support has completed, additionally you can use it for nushell with fish mode.

I don't know detail about nushell but I found that following command could set prompt. But I don't know how to get previous command's exit code...🤔

```sh
# nushell
config set prompt "let COLUMNS = (tput cols);let HOSTNAME = (hostname);messa --error 0 --width $COLUMNS --user $nu.env.USER --host $HOSTNAME --shell fish"
```

**Comment by shosatojp** [comment]
@0323pin I also think it is not possible for nushell, so, you would not need to test it. I keep the staus of nushell support as it is.

Thanks for your issue.
